profiling/paper_timing_tables/timing_comparison.py

```diff
@@ -0,0 +1,338 @@
+"""
+A script for comparing the timing of PINT and TEMPO/Tempo2 for standard fitting 
+cases. Recreates Tables 7, 8, and 9 from PINT paper.
+Requires: 
+    - TEMPO
+    - TEMPO2 
```

Comment:
Do you want to mention that it needs to be run from the same location as its data files? (There are hacks you can use based on `__file__` to look in the same directory as the `.py` file but I don't think it's worth it.)